ethsigner/commandline/src/test/java/tech/pegasys/ethsigner/CommandlineParserTest.java

```diff
@@ -209,7 +228,7 @@ private void missingParameterShowsError(final String paramToRemove) {
   }
 
   @Test
-  public void creatingSignerThrowsDisplaysFailureToCreateSignerText() {
+  void creatingSignerThrowsDisplaysFailureToCreateSignerText() {
```

Comment:
is it throwing or displaying failure text?

Reply:
should be displaying error